versioned_docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md

```diff
@@ -529,7 +529,7 @@ GRANT SHUTDOWN, PROCESS, RELOAD, SUPER, SELECT, REPLICATION CLIENT, REPLICATION
 TO '@MARIADB_REPL_USER@'@'@DATABASE_MASTER_IPADDR@' IDENTIFIED BY '@MARIADB_REPL_PASSWD@';
 ```
 
-Optionally, you can allow these privileges to be used from Central Cluster. It will make some administration scripts runnable from every nodes.
+Optionally, you can allow these privileges to be used from Central Cluster. This will make some administration scripts runnable from every node.
```

Comment:
```suggestion
Optionally, you can allow these privileges to be used from the Central Cluster. This will make some administration scripts runnable from every node.
```